code/controllers/configuration.dm

```diff
@@ -81,6 +81,7 @@
 	var/forumurl = "http://baystation12.net/forums/"
 
 	var/media_base_url = "http://nanotrasen.se/media" // http://ss13.nexisonline.net/media
+	var/overflow_server_url = "byond://nanotrasen.se:6666"

```

Comment:
No clue. this has a default server-specific value to ensure that if everything goes wrong, it just connects to para and doesn't break things any further.